### PR TITLE
Documentation Update to get Volume Snapshots using dellctl

### PR DESCRIPTION
**Pull request by harshitap26**
# Description

- Extend dellctl support to get the volume snapshots for PowerFlex.
- Make the dellctl command generic for the supported platforms while listing the Volumes and Snapshots. 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by falfaroc — APPROVED** [review]

File: content/docs/support/cli/_index.md [hunk 1]
```diff
@@ -31,7 +31,8 @@ This document outlines all dellctl commands, their intended use, options that ca
 | [dellctl encryption rekey](#dellctl-encryption-rekey) | Rekey an encrypted volume |
 | [dellctl encryption rekey-status](#dellctl-encryption-rekey-status) | Get status of an encryption rekey operation |
 | [dellctl images](#dellctl-images) | List the container images needed by csi driver |
-| [dellctl volume get](#dellctl-volume-get) | Gets PowerFlex volume infomation for a given tenant on a local cluster |
+| [dellctl volume get](#dellctl-volume-get) | Gets PowerFlex volume information for a given tenant on a local cluster |
```
Comment (shaynafinocchiaro) [inline content/docs/support/cli/_index.md:34]: Should powerflex be removed from the description if it supported on multiple drivers?

File: content/docs/support/cli/_index.md [hunk 2]
```diff
@@ -31,7 +31,8 @@ This document outlines all dellctl commands, their intended use, options that ca
...
+| [dellctl snapshot get](#dellctl-snapshot-get) | Gets PowerFlex snapshot information for a given tenant on a local cluster |
```
Comment (shaynafinocchiaro) [inline content/docs/support/cli/_index.md:35]: Same as above

File: content/docs/support/cli/_index.md [hunk 3]
```diff
@@ -939,7 +940,7 @@ dellemc/csm-authorization-sidecar:v1.7.0        k8s1.27,k8s1.26,k8s1.25
 
 ### dellctl volume get
 
-Gets PowerFlex volume infomation for a given tenant on a local cluster
+Gets PowerFlex volume information from the authorization proxy for a given tenant on a local cluster
```
Comment (shaynafinocchiaro) [inline content/docs/support/cli/_index.md:943]: Same as above

File: content/docs/support/cli/_index.md [hunk 4]
```diff
@@ -955,22 +956,53 @@ Gets PowerFlex volume infomation for a given tenant on a local cluster
 
 ##### Output
 
-Gets PowerFlex volume infomation for a given tenant on a local cluster. The namespace is the namespace where tenant secret is created. 
+Gets PowerFlex volume information for a given tenant on a local cluster. The namespace is the namespace where tenant secret is created.
```
Comment (shaynafinocchiaro) [inline content/docs/support/cli/_index.md:959]: Same as above

File: content/docs/support/cli/_index.md [hunk 5]
```diff
@@ -955,22 +956,53 @@ Gets PowerFlex volume infomation for a given tenant on a local cluster
...
 # dellctl volume get --proxy <proxy.dell.com> --namespace vxflexos
+NAME                 VOLUME ID          SIZE       POOL    SYSTEM ID          PV NAME          PV STATUS   STORAGE CLASS   PVC NAME                NAMESPACE            SNAPSHOT COUNT
+tn1-k8s-82b35df793   c6c98e30000000d3   8.000000   pool1   636468e3638c840f                                                                                             0
+tn1-k8s-e0e7958ee0   c6cf35ba000001a3   8.000000   pool1   636468e3638c840f   k8s-e0e7958ee0   Bound       vxflexos        pvol-vxflexos           default              2
+tn1-k8s-bc83d4c626   c6cf35c1000001a1   8.000000   pool1   636468e3638c840f   k8s-bc83d4c626   Bound       vxflexos        vol-create-test-xbgnr   snap-test-057de678   3
+```
+
+### dellctl snapshot get
+
+Gets PowerFlex snapshot information from the authorization proxy for a given tenant on a local cluster
```
Comment (shaynafinocchiaro) [inline content/docs/support/cli/_index.md:976]: Same as above

File: content/docs/support/cli/_index.md [hunk 6]
```diff
@@ -955,22 +956,53 @@ Gets PowerFlex volume infomation for a given tenant on a local cluster
...
+```
+  -h, --help                            help for get
+      --insecure optionalBool[=true]    provide flag to skip certificate validation
+      --namespace string                namespace of the secret for the given tenant
+      --proxy string                    auth proxy endpoint to use
+```
+
+##### Output
+
+Get PowerFlex snapshot information for a given tenant on a local cluster. The namespace is the namespace where the tenant secret is created.
```
Comment (shaynafinocchiaro) [inline content/docs/support/cli/_index.md:992]: Same as above

File: content/docs/support/cli/_index.md [hunk 7]
```diff
@@ -955,22 +956,53 @@ Gets PowerFlex volume infomation for a given tenant on a local cluster
 
 ##### Output
 
-Gets PowerFlex volume infomation for a given tenant on a local cluster. The namespace is the namespace where tenant secret is created. 
+Gets the drivers volume information for a given tenant on a local cluster. The namespace is the namespace where tenant secret is created.
 
 >Note: This was output was generated using Authorization Proxy version 1.5.1. Please ensure you are using version 1.5.1 or greater.
```
Comment (shaynafinocchiaro) [inline content/docs/support/cli/_index.md:961]: Is this note still accurate?

Comment (falfaroc) [reply content/docs/support/cli/_index.md:961]: Good point. Those are version that are over a year old. Removed.

**Review by shaynafinocchiaro — APPROVED** [review]

**Pull request merged**